### ISSUE TITLE
Copy-Item alias

### ISSUE THREAD

**Issue by Liturgist** [MicrosoftDocs/PowerShell-Docs#7788]
Copy-Item also has an alias of 'cp' which is more compatible than 'cpi'.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 25c3ba01-933a-ca06-86e3-3eca06c76e93
* Version Independent ID: 9488aaa3-2f5b-d195-c928-93b38146807f
* Content: [Compatibility Aliases - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/learn/compatibility-aliases?view=powershell-7.1)
* Content Source: [reference/docs-conceptual/learn/Compatibility-Aliases.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/learn/Compatibility-Aliases.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @sdwheeler
* Microsoft Alias: **sewhee**

**Comment by doctordns** [comment]
I do not think this is a good idea for an 'official' alias. Adopting it introduces a degree of inconsistency with other `*-Item` cmdlets. In addition to `CPI` we have `GCI` (for `Get-ChildItem`), `RI` (for `Remove-Item`), etc. In each case, the `i` in the alias refers to the word ITEM. I see no justification for adding any more inconsistencies without a very good justification. Besides, I feel it breaks the sacred vow (YMMV).

Ad as ever, you are always free to create your own private aliases and add them to your PowerShell Profile.

**Comment by Liturgist** [comment]
Both PowerShell Core and Windows PowerShell already have `cp` as an alias. Also `dir` for Get-ChildItem and `cd` for Set-Location.

How is a user to know which aliases are "official" and which are not?

Does `cpi` being deemed "official" mean that it will always be supported? If so, why say that aliases should not be used in scripts?

Would it be acceptable to deprecate and drop support for `cp` and `dir` at some point in the future since they are not considered "official?" That seems unlikely to be acceptable.

Does PowerShell include any way to emit an informational message about an alias being used in a script? Is there any "lint-like" tool that would do this?

**Comment by doctordns** [comment]
The alias `CPI` has, IIRC, been a built-in alias for decades (and thanks for reminding me `CP` is too. So you can use whatever alias you like.

The user can always use Get-Alias to discovered the built-0in aliases. So it should be easy to discover And I take issue with the title to this issue that suggests "`cp` is more compatible than `cpi`!".

For me remembering that Item has an alias of i, GCI, GI, RI, CPI all makes perfect sense., REmoving JUST CPI would introduce incompatibilities that are not necessary.

**Comment by Liturgist** [comment]
> The user can always use Get-Alias to discovered the built-0in aliases. So it should be easy to discover And I take issue with the title to this issue that suggests "`cp` is more compatible than `cpi`!".

`cp` has been used by programmers around the world millions, if not billions or trillions, of times more than `cpi`. The "compatibility" to which I refer is what programmers know and not to the new thing.

> For me remembering that Item has an alias of i, GCI, GI, RI, CPI all makes perfect sense., REmoving JUST CPI would introduce incompatibilities that are not necessary.

I am not in any way suggesting that `cpi` be removed. I am asking that `cp` be documented as A) a standard part of the language or B) a transient which should not be relied upon. Which is it?

**Comment by chasewilson** [comment]
Hey @Liturgist, Thanks for the feedback. I do see your point here, however, as it's currrently written, the docs intent is to show the powershell-y alias which would be the capitalized letter of the command and can see the merit in @doctordns point as well. 

I'm not sure I'm sold on how this doc is written though and can see quite a bit of value of rewriting it to include all aliases available within PowerShell. 

I'll do the rewrite and link here for feedback from both you, @Liturgist  and @doctordns as well as @sdwheeler.
